src/integration/pages/idxPage/crossPlatformTests.js

```diff
@@ -0,0 +1,128 @@
+import { runCommonCrossPlatformTests, runMainHeadingTests } from '../../common';
+
+export default () => {
+  runCommonCrossPlatformTests();
+  runMainHeadingTests();
+
```

Comment:
Sorry, just noticed that we are missing the test to check if the `h1` is on the page.

Reply:
This is part of the `mainHeadingTests`